src/lib/libwebaudio.js

```diff
@@ -77,12 +77,7 @@ var LibraryWebAudio = {
 
   // Performs the work of getting the AudioContext's render quantum size.
   $emscriptenGetContextQuantumSize: (contextHandle) => {
-    // TODO: in a future release this will be something like:
-    //   return EmAudio[contextHandle].renderQuantumSize || 128;
-    // It comes two caveats: it needs the hint when generating the context adding to
-    // emscripten_create_audio_context(), and altering the quantum requires a secure
-    // context and fallback implementing. Until then we simply use the 1.0 API value:
-    return 128;
+    return EmAudio[contextHandle].renderQuantumSize || 128;
```

Comment:
Closure will not understand `renderQuantumSize`, unless you update the externs (not sure if the externs of AudioWorklet were managed by us, or by upstream). So this would need to be `EmAudio[contextHandle]['renderQuantumSize']` to be Closure compatible.

I imagine in a distant future when all browsers implement `renderQuantumSize`, we could gate this with `#if MIN_x_VERSION < yyy` checks to phase out the `|| 128` part, and require `renderQuantumSize` to be present.